addons/mrp/security/ir.model.access.csv

```diff
@@ -32,19 +32,19 @@ access_product_supplierinfo_manager,product.supplierinfo user,product.model_prod
 access_mrp_production_manager,mrp.production manager,model_mrp_production,mrp.group_mrp_manager,1,0,0,0
 access_mrp_bom_stockuser,mrp.bom,model_mrp_bom,stock.group_stock_user,1,0,0,0
 access_mrp_bom_line_stockuser,mrp.bom.line,model_mrp_bom_line,stock.group_stock_user,1,0,0,0
-access_uom_uom_mrp_manager,uom.uom mrp_manager,uom.model_uom_uom,mrp.group_mrp_manager,1,1,1,1
-access_product_category_mrp_manager,product.category mrp_manager,product.model_product_category,mrp.group_mrp_manager,1,1,1,1
-access_product_template_mrp_manager,product.template mrp_manager,product.model_product_template,mrp.group_mrp_manager,1,1,1,1
-access_product_product_mrp_manager,product.product mrp_manager,product.model_product_product,mrp.group_mrp_manager,1,1,1,1
-access_product_pricelist_mrp_manager,product.pricelist mrp_manager,product.model_product_pricelist,mrp.group_mrp_manager,1,1,1,1
+access_uom_uom_mrp_manager,uom.uom mrp_manager,uom.model_uom_uom,mrp.group_mrp_manager,1,0,0,0
+access_product_category_mrp_manager,product.category mrp_manager,product.model_product_category,mrp.group_mrp_manager,1,0,0,0
+access_product_template_mrp_manager,product.template mrp_manager,product.model_product_template,mrp.group_mrp_manager,1,0,0,0
+access_product_product_mrp_manager,product.product mrp_manager,product.model_product_product,mrp.group_mrp_manager,1,0,0,0
+access_product_pricelist_mrp_manager,product.pricelist mrp_manager,product.model_product_pricelist,mrp.group_mrp_manager,1,0,0,0
 access_product_group_res_partner_mrp_manager,res_partner group_mrp_manager,base.model_res_partner,mrp.group_mrp_manager,1,1,1,0
 access_product_pricelist_item_mrp_manager,product.pricelist.item mrp_manager,product.model_product_pricelist_item,mrp.group_mrp_manager,1,1,1,1
-access_product_tag_mrp_manager,product.tag.mrp.manager,product.model_product_tag,group_mrp_manager,1,1,1,1
+access_product_tag_mrp_manager,product.tag.mrp.manager,product.model_product_tag,group_mrp_manager,1,0,0,0
 access_resource_calendar_manufacturinguser,resource.calendar manufacturing.user,resource.model_resource_calendar,mrp.group_mrp_user,1,0,0,0
 access_mrp_unbuild,mrp.unbuild,model_mrp_unbuild,group_mrp_user,1,1,1,1
 access_mrp_unbuild_manager,mrp.unbuild manager,model_mrp_unbuild,group_mrp_manager,1,1,1,1
 access_mrp_document_mrp_manager,product.document group_user,model_product_document,group_mrp_manager,1,1,1,1
-access_mrp_document_mrp_user,product.document group_user,model_product_document,group_mrp_user,1,1,1,1
+access_mrp_document_mrp_user,product.document group_user,model_product_document,group_mrp_user,1,0,0,0
```

Comment:
There's still `access_product_document_user`, which makes this change obsolete :thinking:

![image](https://github.com/user-attachments/assets/06eed12b-48c7-4611-991f-9c40d69adfa9)

Reply:
Oops, I forgot to remove the internal one, I have updated it now :)